typescript/src/app/app.module.ts

```diff
@@ -2,10 +2,12 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppComponent } from './app.component';
+import { NewcompComponent } from './newcomp/newcomp.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7541583)
```suggestion
import { NewcompComponent } from "./newcomp/newcomp.component";
```